extra-files/galaxy-kickstart/welcome.html

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+<head>
+    <title>Homepage</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <link rel="stylesheet" href="style/base.css" type="text/css" />
+</head>
+<body>
+    <div class="document">
+        <p align="center">
+            <img src="images/galaxy-kickstart_logo.png" width="400" />
+        </p>
+    </div>
+    <div class="container">
```

Comment:
Can we mention the release and changeset revision somewhere in the html?
If you figure out a nice tag, we can maintain the welcome.html as a template.